### ISSUE TITLE
Using FCM token for iOS and Android

### ISSUE THREAD

**Issue by skyplor** [smilefam/SendBird-SDK-JavaScript#82]
Hi guys, so previously we were using [react-native-fcm](https://github.com/evollu/react-native-fcm) and getting the respective APNS & FCM token for the different type of devices iOS & Android respectively. Things are good in configuring Sendbird as we register iOS devices using their APNS token and Android devices using their FCM token. And our devices were able to receive push notifications in all states (foreground, background, closed).

But we decided to switch over to [react-native-firebase](https://github.com/invertase/react-native-firebase) but this library handles the iOS and Android devices with FCM tokens. We provide the corresponding APN key in the firebase console itself.

So instead of APNS token, now we register both iOS & Android devices with FCM token. But this broke the push notification for iOS devices when the devices are in the **background & closed** states. Foreground state still works.

How can we resolve this part?

**Comment by skyplor** [comment]
Hi @chrisallo thanks for your reply. Because [react-native-fcm](https://github.com/evollu/react-native-fcm) is stopping its maintenance and is recommending react-native-firebase library, we moved on to using that. But [react-native-firebase](https://github.com/invertase/react-native-firebase/) does not support or provide a way for us to get the APNS token.

Nonetheless, we have since added OneSignal to help manage the push notifications and are able to get the APNS / FCM token from there, so all is good for us to use Sendbird =)

Here's my working env:
react-native: 0.55.4
sendbird: 3.0.66
